1.4.2-beta.4 is faulty

> Seems that node-fetch was upgraded again on the PR #840 and this issue is back.  

> Going back to the previous beta (1.4.2-beta.3) work as expected.

_Originally posted by @Taulim in https://github.com/intersystems-community/vscode-objectscript/issues/791#issuecomment-1021162446_

1.4.2-beta.4 was the product of a dependabot security bump of node-fetch from 3.0.0 to 3.1.1

ObjectScript output channel reports:
```
ntersystems-community.vscode-objectscript version 1.4.2-beta.4 activating with proposed APIs available.

Cannot find module 'node:http'
Require stack:
- c:\Users\xxx\.vscode-insiders\extensions\intersystems-community.vscode-objectscript-1.4.2-beta.4\dist\extension.js
- c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\vs\loader.js
- c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\bootstrap-amd.js
- c:\Users\xxx\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\out\bootstrap-fork.js
Check your server details in Settings ...
```

--- a/package.json
+++ b/package.json
@@ -1189,7 +1189,7 @@
     "glob": "^7.1.6",
     "iconv-lite": "^0.6.0",
     "mkdirp": "^1.0.4",
-    "node-fetch": "3.1.1",
+    "node-fetch-cjs": "3.1.1",
     "vscode-cache": "^0.3.0",
     "vscode-debugadapter": "^1.41.0",
     "vscode-debugprotocol": "^1.41.0",

--- a/src/api/index.ts
+++ b/src/api/index.ts
@@ -1,8 +1,6 @@
-// import fetch from "node-fetch";
-// mod.cjs
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-const fetch = (...args) => import("node-fetch").then(({ default: fetch }) => fetch(...args));
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { default: fetch } = require("node-fetch-cjs");
+
 import * as httpModule from "http";
 import * as httpsModule from "https";
 import * as url from "url";

